runtime/j9vm/java11vmi.c

```diff
@@ -890,8 +911,25 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
 #endif /* JAVA_SPEC_VERSION >= 21 */
 							vm->runtimeFlags |= J9_RUNTIME_JAVA_BASE_MODULE_CREATED;
 							Trc_MODULE_defineModule(currentThread, "java.base", j9mod);
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+							if (J9_ARE_ANY_BITS_SET(vm->checkpointState.flags, J9VM_CRAC_IS_CHECKPOINT_ENABLED)) {
+								UDATA retval = ERRCODE_GENERAL_FAILURE;
```

Comment:
i think its better to do this at the Java level via `Module::addExports` so if there are changes in the future we dont need to think about this case

Reply:
Moved the `addExports` into java.